src/components/components/NavbarMobileLogoBar/NavbarMobileLogoBar.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import logo from "../Navbar/assets/skillChainLogoLight.svg";
+import "./NavbarMobileLogoBar.scss";
+
+function NavbarMobileLogoBar({ handleClick, isClicked }) {
```

Comment:
I don't like the naming of this component. Maybe Rename this one to `MobileNavigationMenu` and the other to `MobileNavigationLinks` or something like that?

Reply:
makes perfectly sense, accepted, thank you!